MiniApp/Classes/JavascriptBridge/bridge.js

```diff
@@ -114,6 +114,22 @@ MiniAppBridge.requestPermission = function(permissionType) {
     );
   });
 };
+
+/**
+ Below code will override the navigator.geolocation.getCurrentPosition method
+ */
+navigator.geolocation.getCurrentPosition = function(success, error, options) {
```

Comment:
The `navigator.geolocation` shouldn't be modified on Android, so this should be set within `if (isPlatform.iOS())`.

Reply:
Done